src/P_mat/elemental_em.jl

```diff
@@ -13,46 +13,53 @@ export identity_eem, create_id_eem, fixed_ones_eem, ones_eem, one_size_bloc
     Elemental_em{T} <: DenseEltMat{T}
 
 Represent an elemental element-matrix.
-`indices` retains the indices of the elemental variables.
-`nie` is the elemental size (`=length(indices)`).
-`Bie` a `Symmetric{T, Matrix{T}}`.
-`counter` counts how many update the elemental matrix goes through from its allocation.
+It has fields:
+
+* `indices`: indices of elemental variables;
+* `nie`: elemental size (`=length(indices)`);
+* `Bie`: the elemental matrix`::Symmetric{T, Matrix{T}}`;
```

Comment:
```suggestion
* `Bie::Symmetric{T, Matrix{T}}`: the elemental matrix;
```
I find it clearer if you indicate types this way.